src/components/MenuBar.vue

```diff
@@ -303,6 +305,17 @@ export default defineComponent({
       );
     };
 
+    const handleDeactivateMenu = (id?: string) => {
```

Comment:
@sunhwan this id can be removed if not used

Reply:
Thanks, It is done!